docs/bookstore/page-0.5/environment-setup.md

```diff
@@ -270,7 +270,7 @@ node-server-svc          LoadBalancer   10.101.90.35    <pending>     80:31792/T
 If port forwarding is required, open a new terminal and run:
 
 ```shell
-kubectl port-forward svc/node-server-svc 8080:80
+kubectl port-forward svc/node-server-svc 8080:8080
```

Comment:
This change doesn't seem relevant to this PR, can you remove it?

Reply:
Oops. Done, thanks!